src/App.tsx

```diff
@@ -69,6 +75,19 @@ export const App = () => {
     }
   }, [isNativeOnboardOpen, openNativeOnboard, showWelcomeModal])
 
+  const portfolioWalletId = useAppSelector(selectWalletId)
+
+  const {
+    state: { deviceId: contextDeviceId },
+  } = useWallet()
+
+  useEffect(() => {
+    console.log({
+      portfolioWalletId,
+      contextDeviceId,
+    })
+  }, [contextDeviceId, portfolioWalletId])
+
```

Comment:
TODO: remove before merge. Used to check alignment between redux and redux context. 

Note that differences do occur:
- redux is `undefined` and provider is defined -> OK ✅
- redux is a string and provider is the SAME string -> OK ✅

If you see these differences:
- redux is defined and provider is `undefined` -> NOK ❌
- redux is a string and provider is a DIFFERENT string -> NOK ❌